src/mlpack/tests/recurrent_network_test.cpp

```diff
@@ -690,13 +698,23 @@ void DistractedSequenceRecallTestNetwork(
     // prompts.
     for (size_t i = 0; i < testDistractedSequenceCount; i++)
     {
-      arma::mat output;
-      arma::mat input = testInput.at(0, i);
+      arma::cube output;
+      arma::cube input(testInput.at(0, i).memptr(), inputSize, 1,
+          testInput.at(0, i).n_elem / inputSize, false, true);
 
       model.Predict(input, output);
-      data::Binarize(output, output, 0.5);
+      for (size_t j = 0; j < output.n_slices; ++j)
+      {
+        arma::mat outputSlice = output.slice(j);
+        data::Binarize(outputSlice, outputSlice, 0.5);
```

Comment:
Do you think we should add ``cube`` support, might be helpful to postprocess the data. If you agree, this could be a good beginner task.

Reply:
Yeah, I think it would be a nice beginner task to add.  I'll go ahead and write up an issue for it.  I think no new code is necessary; just the existing `Binarize()` can be adapted for this case, then this code can be simplified.